docs/api/05-language-reference.md

```diff
@@ -1215,9 +1215,74 @@ The following features are not yet implemented, but we are planning to add them
 * `await`/`defer` statements - see https://github.com/winglang/wing/issues/116 to track.
 * Promise function type - see https://github.com/winglang/wing/issues/1004 to track.
 
-### 1.14 Roadmap
+[`▲ top`][top]
+
+---
+
+### 1.14 Bytes type
+
+The `bytes` and `mutbytes` types store immutable and mutable sequences of binary data.
+Under the hood, these types are implemented using JavaScript's `Uint8Array` type.
+`mutbytes` allows for bytes to be changed in-place, but it cannot be resized.
```

Comment:
originally I wrote this up with just the `bytes` type, but inevitably I had to ask whether the data type should be immutable or not -- to align with the rest of the language, I suggest here separating it into two separate types. they could also be named `Bytes` and `MutBytes` though it might be worth lower-casing them eventually per https://github.com/winglang/wing/issues/1468

Reply:
+1 on the lowercase (for all types).

Reply:
Not sure a mutable version is p1

Reply:
Agree